environment/frontend_server/temp_storage/curr_sim_code.json

```diff
@@ -1,3 +1,3 @@
 {
-  "sim_code": "33"
-}
+  "sim_code": "so-2-s-3-599-799"
+}
```

Comment:
Looks like your latest commit changed this file. Can you change it back, please?